vsts/gatedBuild.ps1

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+Function IsWindows() 
+{
+	return ([Environment]::OSVersion.Platform -eq [System.PlatformID]::Win32NT)
+}
+
+if (isWindows) 
+{
+	Write-Host Start ETL logging
+	logman create trace IotTrace -o iot.etl -pf tools/CaptureLogs/iot_providers.txt
+	logman start IotTrace
+}
+
+Write-Host List active docker containers
+docker ps -a
+
+#Load functions used to check what, if any, e2e tests should be run
+. .\vsts\determine_tests_to_run.ps1
+
+$runTestCmd = ".\build.ps1 -clean -build -configuration DEBUG -framework $env:FRAMEWORK"
+if (IsPullRequestBuild)
+{
+	Write-Host "Pull request build detected, will run pr tests"
+	$runTestCmd += " -prtests"
+
+	if (ShouldSkipDPSTests) 
+	{
+		Write-Host "Will skip DPS tests"
+		$runTestCmd += " -skipDPSTests"
+	}
+	else 
+	{
+		Write-Host "Will run DPS tests"
+	}
+	
+	if (ShouldSkipIotHubTests) 
+	{
+		Write-Host "Will skip Iot Hub tests"
+		$runTestCmd += " -skipIoTHubTests"
+	}
+	else 
+	{
+		Write-Host "Will run Iot Hub tests"
+	}	
+	
+	#if (ShouldSkipPnPTests) 
```

Comment:
uncomment this

Reply:
This is deliberately commented out. We want to skip all PnP tests for now, regardless of what changes are made

Reply:
Are you sure? I remember updating the pipeline manually a couple of days pack to pick up PnP E2E tests
@prmathur-microsoft  to confirm.

Reply:
Did we mean that we should disable the tests that are failing/ all PnP E2E tests?